### DISCUSSION TITLE
Process for wrong completes?

### DISCUSSION THREAD

**Discussion by bannsec** [community/community#6718] [Copilot Conversations]
Is there a process people should go through for incorrect completions? For instance, I recently was testing copilot's ability to utilize python crypto libraries to perform handshakes and encryption and literally __none__ of the 10 suggestions would work. Any further prompting basically failed as well since it kept going back to what appears to be obsoleted and removed library calls.

There were other examples (such as a lot of the Windows API calls being wrong), but in general I'm guessing there should be some way to flag that copilot didn't provide good feedback.

**Comment by g2rant** [comment]
Also interested if I hit <tab> too soon and unintentionally accepted something I didn't want, or maybe I accepted something that is *close* to what I want after a couple of small edits, whether there's any way (or value) for Copilot to recognize this as feedback.

**Comment by Urethramancer** [comment]
Sometimes writing a verbose comment helps. I've foind many cases where I get nothing useful though, like it suggesting a comment instead of actual code, or just returning nothing. I still haven't managed to get it to use the GPG packages for Go, and always get code which calls out to the CLI tool.

Outdated packages have been a problem for me too, but at least for the AWS SDK that seems to be a problem _everywhere_. When I search for docs, I get the v1 examples. When I let Go's goimports/gofmt pick packages, I get v1. Copilot suggestions likewise.

And most annoyingly, at one point yesterday it kept adding "&" in front of a variable that shouldn't be a pointer at all. I want that "this is awful" button to click for feedback too :)
